### ISSUE TITLE
HA restart- 10.0.2-nas2

### ISSUE THREAD

**Issue by cokeman0** [dianlight/hassio-addons#159]
### Addon

SambaNAS

### Description

Sorry not fixed in nas2

### Reproduction steps

```bash
restart (not reboot) HA
```

### Addon Logs

```bash
noting shown
```

### Addon Config

```bash
clean, no special setup
```

**Comment by dianlight** [comment]
I need the config and the architecture to reproduce.

**Comment by cokeman0** [comment]
Sure.

Lenovo ThinkCentre M700 Tiny | Intel 6th Gen | i5-6500T | 8 GB
Version | core-2023.6.3
Installation Type | Home Assistant OS
Development | false
Supervisor | true
Docker | true
User | root
Virtual Environment | false
Python Version | 3.11.4
Operating System Family | Linux
Operating System Version | 6.1.34
CPU Architecture | x86_64

config:
```
workgroup: EDB
username: REDACTED
allow_hosts:
  - 172.30.33.0/12
  - 192.168.0.112
  - 192.168.0.1
  - 192.168.0.157
automount: true
moredisks:
  - DataDisk
available_disks_log: false
medialibrary:
  enable: false
  ssh_private_key: <YOU SUPER SECRET PRIVATE KEY>
veto_files:
  - ._*
  - .DS_Store
  - Thumbs.db
  - icon?
  - .Trashes
compatibility_mode: false
wsdd2: false
mqtt_use_legacy_entities: false
autodiscovery: {}
other_users: []
acl: []
interfaces: []
password: REDACTED
mqtt_enable: true
enable_smart: true
log_level: error
hdd_idle_seconds: 600

```

**Comment by cokeman0** [comment]
Any update on this

**Comment by dianlight** [comment]
> Any update on this

I can't reproduce un my system but I found some time critical concurrency in startup script that can be the source. I'm now very busy on another project but I will release the update in short time.

**Comment by cokeman0** [comment]
Super. Thanks..

Med venlig hilsen

Michael Dahl

ons. 6. sep. 2023 18.49 skrev Lucio Tarantino ***@***.***>:

**Comment by dianlight** [comment]
should be fixed  in the release 12.1.0-nas
